app/assets/stylesheets/modules/profile.scss

```diff
@@ -0,0 +1,13 @@
+
+.circular img {
+  border-radius: 50%;
+  height:200px;
+  margin:25px;
+  width: 200px;
```

Comment:
Properties should be ordered border-radius, height, margin, transform, transition, width